epfast-annotations/src/main/java/com/exactpro/epfast/CreatorService.java

```diff
@@ -7,7 +7,19 @@
 import static com.exactpro.epfast.util.Stream.streamIterable;
 
 public class CreatorService {
-    public static Collection<ICreator> providers() {
+    private static Collection<ICreator> providers;
```

Comment:
you can just add ` = getProviders()` and remove the following static initialization block